articles/azure-monitor/app/java-standalone-profiler.md

```diff
@@ -22,7 +22,7 @@ The Application Insights Java Profiler provides a system for:
 
 The Application Insights Java profiler uses the JFR profiler provided by the JVM to record profiling data, allowing users to download the JFR recordings at a later time and analyze them to identify the cause of performance issues.
 
-This data is gathered on demand when trigger conditions are met. The available triggers are thresholds over CPU usage and Memory consumption.
+This data is gathered on demand when trigger conditions are met. The available triggers are thresholds over CPU usage and Memory consumption, and Request (SLA triggers).
```

Comment:
I would probably be a bit more specific as many may not know what is meant by "SLA triggers". So might split it up into an extra sentence, i.e:
```
This data is gathered on demand when trigger conditions are met. The available triggers are thresholds over CPU usage and Memory consumption, and Request (SLA triggers). Request triggers monitor Spans generated by Open Telemetry and allow the user to configure SLA requirements over the duration of those Spans.
```